packages/contracts-bedrock/.gas-snapshot

```diff
@@ -17,7 +17,7 @@ AttestationStationTest:test_attest_bulk_succeeds() (gas: 703749)
 AttestationStationTest:test_attest_individual_succeeds() (gas: 632087)
 AttestationStationTest:test_attest_single_succeeds() (gas: 651325)
 BlockOracle_Test:test_checkpointAndLoad_succeeds() (gas: 58341)
-BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 12805)
+BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 13659)
```

Comment:
tried updating the gas snapshot, but the gas changes make no sense to me. Is there a specific check I can do to fix this?

Reply:
I am guessing you are using the wrong version of forge

Reply:
Can conffirm, my bad. Running `pnpm update:foundry` worked